### PR TITLE
axum-core: Version 0.2.3

### PR DESCRIPTION
**Pull request by davidpdrsn**
Gonna do one for axum when this is out.

### CODE REVIEW

**Comment by davidpdrsn** [comment]
@jplatte ping :)

**Comment by jplatte** [comment]
Oh, failed to submit my review comment 🤦🏼

File: axum-core/Cargo.toml [hunk 1]
```diff
@@ -8,7 +8,7 @@ license = "MIT"
 name = "axum-core"
 readme = "README.md"
 repository = "https://github.com/tokio-rs/axum"
-version = "0.2.2"
+version = "0.2.3" # remember to also bump the version that axum depends on
```
[the remark this answers is not in the document]
Comment (davidpdrsn) [reply axum-core/Cargo.toml:11]: Doing a separate release of axum next.

[the remark this answers is not in the document]
Comment (jplatte) [reply axum-core/Cargo.toml:11]: But the version of axum-core in axum's dependencies can be bumped now, then you can't forget to do it later..

[the remark this answers is not in the document]
Comment (davidpdrsn) [reply axum-core/Cargo.toml:11]: Oh yeah of course! Done!

**Review by jplatte — APPROVED** [review]

**Pull request merged**